### ISSUE TITLE
Please continue developing! [FEAT]

### ISSUE THREAD

**Issue by spiljj** [tefkah/zotero-night#179]
The nord theme is so great for zotero, the native dark theme is not a match, maybe develop as a them for zotero?

**Comment by medium-old** [comment]
Yes please! Nord theme for dark PDF was instrumental for me picking Zotero. The low contrast is incredibly easier for reading. Wondering if there's any way to gain momentum on this issue.
